src/DssSpell.sol

```diff
@@ -136,8 +148,15 @@ contract DssSpellAction is DssAction {
         DssExecLib.setIlkStabilityFee("WSTETH-B", THREE_PT_FOUR_NINE, /* doDrip = */ true);
 
         // --- Spark Protocol Parameter Changes ---
-        // Poll: https://vote.makerdao.com/polling/QmWatYqy#poll-detail
+        // D3M Parameter Adjustments Poll: https://vote.makerdao.com/polling/QmWatYqy#poll-detail
+        // Executive Proxy Poll: https://vote.makerdao.com/polling/Qmc9fd3j#poll-detail
+        // Onboard rETH Poll: https://vote.makerdao.com/polling/QmeEV7ph#vote-breakdown (Inside Proxy Spell)
+        // DAI Interest Rate Strategy Poll: https://vote.makerdao.com/polling/QmWodV1J#poll-detail (Inside Proxy Spell)
         // Forum: https://forum.makerdao.com/t/2023-05-24-spark-protocol-updates/20958
+        DssExecLib.setIlkAutoLineParameters("DIRECT-SPARK-DAI", /* line */ 20 * MILLION, /* gap */ 20 * MILLION, /* ttl */ 8 hours);
+        DssExecLib.authorize(SPARK_PROXY, DssExecLib.esm());
```

Comment:
if we're authorizing `SPARK_PROXY` on ESM, shouldn't it be added to chainlog? I would then suggest to name it as `SUBDAO_PROXY_SPARK` to not confuse with other proxies in the chainlog and be able to filter them in the future

Reply:
Good point - yeah this makes sense.